src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -40,7 +40,7 @@ def __init__(
         ws: WorkspaceClient,
         backend: SqlBackend,
         table_mapping: TableMapping,
-        migration_status_refresher: MigrationStatusRefresher,
+        migration_status_refresher: TableMigrationStatusRefresher,
```

Comment:
@asnare : Could you rename this parameter and the attribute it is assigned to below accordingly to the class rename?